mlflow/utils/requirements_utils.py

```diff
@@ -330,6 +330,9 @@ def _capture_imported_modules(model_uri, flavor):
     "databricks.automl_runtime": ["databricks-automl-runtime"],
     "databricks.model_monitoring": ["databricks-model-monitoring"],
 }
+MLFLOW_MODULES_TO_PACKAGES = {
+    "mlflow.gateway": ["mlflow[gateway]"],
```

Comment:
the value here is kind of arbitrary—it doesn't really get used except in `requirements_utils.py:429` but it's hardcoded there too. maybe i should const it?